resources/views/filament/app/pages/survey-dashboard.blade.php

```diff
@@ -0,0 +1,174 @@
+                </div>
+                <!-- White Section -->
+                <div class="w-3/4 bg-white flex p-4">
+                    <div class="w-1/2 pr-4 border-r border-gray-300">
+                        <h3 class="text-black font-bold mb-2">Add or manage additional data</h3>
+                        <p class="text-gray-600 mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vehicula efficitur metus, id fermentum urna volutpat in.</p>
+                        <div class="text-center mb-4">
+                            <a href="{{ url(AddData::getUrl()) }}" class="bg-blue text-white py-2 px-4 rounded-full hover-effect">
+                                VIEW AND UPDATE
+                            </a>
+                        </div>
+                    </div>
+                    <div class="w-1/2 pl-4">
+                        <h3 class="text-black font-bold mb-2">Survey languages</h3>
+                        <p class="text-gray-600 mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vehicula efficitur metus, id fermentum urna volutpat in.</p>
+                        <div class="text-center">
+                            <a href="{{ url(SurveyLanguages::getUrl()) }}" class="bg-blue text-white py-2 px-4 rounded-full hover-effect">
+                                VIEW AND UPDATE
+                            </a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <!-- Sampling card -->
+            <div class="flex shadow-md overflow-hidden md:col-span-2 h-80">
+                <div class="w-1/4 bg-green text-white text-lg font-bold flex flex-col items-center justify-center p-4">
+                    <img src="/images/sampling_icon.png" alt="Sampling Icon" class="w-10 h-10 mb-2">
+                    <span class="mt-2 text-center">Sampling</span>
+                </div>
+                <div class="w-3/4 bg-white p-4">
+                    <h3 class="text-black font-bold mb-2">Sampling frame</h3>
+                    <p class="text-gray-600 mb-4">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam vehicula efficitur metus, id fermentum urna volutpat in.</p>
+                    <div class="text-center">
+                        <a href="{{ url(Sampling::getUrl()) }}" class="bg-blue text-white py-2 px-4 rounded-full hover-effect">
+                            VIEW AND UPDATE
+                        </a>
+                    </div>
+                </div>
+            </div>
+
+            <!-- Localisation card -->
+            <div class="flex shadow-md overflow-hidden md:col-span-4 h-80">
+                <!-- Green Section -->
+                <div class="w-2/8 bg-green text-white text-lg font-bold flex flex-col items-center justify-center p-4">
+                    <img src="/images/localisation_icon.png" alt="Localisation Icon" class="w-10 h-10 mb-2">
```

Comment:
Never force both a height and a width on an image unless you know those are the exact proportions of the image - otherwise you will get a squashed image, which is up near the top of things to avoid in design: 


**Squashed:**
![CleanShot 2024-10-31 at 10 25 43](https://github.com/user-attachments/assets/d34f3835-aa68-405e-a9bd-1f07e6e133b3)

**Unsquashed**:
![CleanShot 2024-10-31 at 10 30 00](https://github.com/user-attachments/assets/30bbfab5-00ac-4680-939e-e61ed088f6b6)